platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -59,6 +59,7 @@ private static String version(String module) {
 		throw new AssertionError("module name is unknown: " + module);
 	}
 
+	@SuppressWarnings("FilesLinesLeak") // stream closed indirectly by JUnit 5
```

Comment:
The source of the warning is the `Files.lines(...)...` chain.

error-prone is not intelligent enough to recognise that the Stream returned from `Files.lines(...)...`, a Stream which requires closing by the user to free up filesystem resources and which is being passed outside of the enclosing method (usually a bad thing), will eventually be closed by JUnit 5 itself,